### DISCUSSION TITLE
Bölenleri Say

### DISCUSSION THREAD

**Discussion by OmerB2** [sonsuzus/sonsuzus#349] [Soru Cevap]
**Problem**
Size 3 tamsayı verildi - l, r ve k. l ve r (her ikisi de dahil) arasındaki kaç sayının k ile bölünebileceğini bulun. Bu sayıları yazdırmanıza gerek yok, sadece sayılarını bulmanız yeterli.

**Girdi Formatı:**

Girdinin ilk ve tek satırı boşlukla ayrılmış 3 tamsayı içerir _l_ , _r_ ve _k_.

**Çıktı Formatı:**

Gerekli yanıtı tek bir satıra yazdırın.

**Kısıtlamalar**

_1 ≤ l ≤ r ≤ 1000
1 ≤ k ≤ 1000_

```
Örnek Girdi
1 10 1

Örnek Çıktı
10
```

**Comment by tolga0102** [comment]
```python
l,r,k=input("Sayıları birer boşluk bırakarak girin. ").split()
bolunen=0
for a in range(int(l),int(r)+1):
    if a%int(k)==0:
        bolunen+=1
print(l,"ve",r,"sayıları arasında",bolunen,"sayı",k,"sayısına bölünebilir.")
```
